department-of-veterans-affairs/vets-api
# Submitting a 1995-STEM form results in a 1995 form being saved #6096

## Description of change
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6096

The issue is that when signed in and going through at 1995 form at `/education/apply-for-education-benefits/application/1995/` it initially creates a in progress form using the `22-1995` as form id however on submission of form it is possible that the `claim.form_id` is actually `22-1995S` as decided by choices the user has made on stem and stem eligibility pages.  

## Testing done
1. Access `/education/apply-for-education-benefits/application/1995/introduction`
2. Log in and begin a new application
3. Fill in all required fields on the applicant information page and click continue
4. Select either of the Chapter 33 benefits and click continue
5. Select Yes for "Are you applying for the Edith Nourse Rogers STEM Scholarship?(*Required)" and Yes for "Are you enrolled in an undergraduate STEM degree program?(*Required)"
6. Complete the rest of the application and submit
7. After a successful submission, ensure you are still logged in and access `education/apply-for-education-benefits/application/1995/introduction` again

## Testing planned


## Acceptance Criteria (Definition of Done)

#### Unique to this PR

- [ ] _Replace this line with individual tasks unique to your PR_

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)
